Misc/NEWS.d/next/Library/2020-05-08-15-48-39.bpo-40503.elZyxc.rst

```diff
@@ -0,0 +1 @@
+PEP 615, the zoneinfo module. Adds support for the IANA time zone database.
```

Comment:
You should move this news entry to the Documentation section. Changes to add links:

```suggestion
:pep:`615`, the :mod:`zoneinfo` module. Adds support for the IANA time zone database.
```

Reply:
Why the documentation section? This is the news entry for `zoneinfo`, not the news entry for "added documentation for `zoneinfo`"